.gitignore

```diff
@@ -29,6 +29,7 @@
 /dist-assets/sslocal
 /dist-assets/sslocal.exe
 /dist-assets/shell-completions/
+/gui/scripts/mi_proto/
```

Comment:
I guess we could remove this as well now when the readme is referring to a tmp-directory?

Reply:
Done